packages/core/src/shared/types.ts

```diff
@@ -0,0 +1,16 @@
+import { GraphQLResolveInfo } from 'graphql';
+
+export type ID = string;
```

Comment:
Maybe we can use https://github.com/pirix-gh/ts-toolbelt#where-to-start as devDependency and avoid some of those?